src/cljs/main/broadfcui/page/method_repo/create_method.cljs

```diff
@@ -37,6 +37,9 @@
   {:component-will-mount
    (fn [{:keys [props locals]}]
      (swap! locals assoc :info (build-info props)))
+   :get-initial-state
+   (fn []
+     {:file-input-key (gensym "wdl-uploader-")})
```

Comment:
Do we need this? Is it a problem if there's no key the first time, and then one gets created afterwards? Not a big deal either way.

Reply:
Will double check - but possibly not.

Reply:
It does work without this - will remove.